### ISSUE TITLE
App is trying to generate previews for pictures.

### ISSUE THREAD

**Issue by awptechnologies** [nextcloud/previewgenerator#526]
After installing this app and running the initial command in cli before making cron job i see it is generating previews for all the images in my nextcloud. Is this normal? in the web interface i can see my picture thumbnails fine i wanted this app to create thumbnails/previews for videos. 

What exactly is the app doing to all my pictures?

**Comment by st3iny** [comment]
As documented in the readme, the command `preview:generate-all` will generate previews for **all** existing files.

If you want to generate previews for changed files only (starting from the instant you installed the app), please run `preview:pre-generate` in a cron job.

It is currently not possible to only generate previews for certain file types only, e.g. video files. Please have a look [here](https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/config_sample_php_parameters.html#enabledpreviewproviders) for more information on how to set up preview generation for video files.

**Comment by awptechnologies** [comment]
If i run preview:generate-all it will just overwrite existing previews right? also with ffmpeg installed it should allow me to preview videos as well correct?

**Comment by st3iny** [comment]
No, it will not overwrite existing previews. Only the previews that are not yet existing will be generated.

To overwrite/replace existing previews, you would need to delete the old ones first. Please have a look [here](https://github.com/nextcloud/previewgenerator?tab=readme-ov-file#i-want-to-resetregenerate-all-previews) for more information.

**Comment by st3iny** [comment]
> also with ffmpeg installed it should allow me to preview videos as well correct?

Only if you enable the corresponding movie preview provider. It is disabled by default due to security concerns.

E.g.:

```php
'enabledPreviewProviders' => [
        // Defaults:
        'OC\Preview\BMP',
        'OC\Preview\GIF',
        'OC\Preview\JPEG',
        'OC\Preview\Krita',
        'OC\Preview\MarkDown',
        'OC\Preview\MP3',
        'OC\Preview\OpenDocument',
        'OC\Preview\PNG',
        'OC\Preview\TXT',
        // Video files via ffmpeg:
        'OC\Preview\Movie',
],
```
